src/components/recruit/editor/table/tableUtils.ts

```
@@ -0,0 +1,71 @@
+}
+
+/** 행 이동 (헤더 보호) */
+export function moveTableRow(editor: Editor, fromIndex: number, toIndex: number): boolean {
+  const info = getTableInfo(editor);
+  if (!info) return false;
+  const { tableStart, tableNode, rows } = info;
+  if (fromIndex === toIndex || fromIndex < 1 || toIndex < 1 || fromIndex >= rows || toIndex >= rows)
+    return false;
+
+  const rowNodes: PmNode[] = [];
+  tableNode.forEach((row) => rowNodes.push(row));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)